microcontroller_lab.rst

```diff
@@ -0,0 +1,54 @@
+================
+Microcontrollers
+================
+
+In this lab, you will build circuits which are controlled by the Arduino microcontroller.
+
+.. conetents::
```

Comment:
Check spelling here. Remember you can type `make` to build this page for yourself.